test/controllers/add.test.js

```diff
@@ -0,0 +1,35 @@
+const Botmock = require('botkit-mock');
+const {expect, assert} = require("chai");
```

Comment:
We are only using the `expect` function down below, so let's drop the `assert` function from being imported.

Reply:
no prob.